client/src/containers/App/App.js

```diff
@@ -6,8 +6,8 @@ import { Outlet } from 'react-router';
  * Empty container for the moment, will eventually contain the CMS menu`
  * and apply to document.body, rather than just one specific DOM element.
  */
-const App = () => (
-  <div className="app"><Outlet /></div>
+const App = ({ children }) => (
+  <div className="app">{children}<Outlet /></div>
```

Comment:
We have to have the blocker as a child of `App` because it has to be a child of the `RouteProvider` and this is the only component we know for sure will be in there.
But because this replaceable via the `reactRouteRegister` we don't even know that the root component will specifically be `App` so we can't just slap it directly here, we have to pass it in as a child prop.